openapi.yaml

```
@@ -2269,3 +2939,9 @@ components:
             read:ssps: read your ssps
             write:parties: modify parties in your account
             read:parties: read your parties
+            write:poams: modify poams in your account
+            read:poams: read your poams
+            write:assessmentPlan: modify assessment plans in your account
+            read:assessmentPlan: read your assessment plans
```

```suggestion
            write:assessmentPlans: modify assessment plans in your account
            read:assessmentPlans: read your assessment plans
```

Fix here and wherever else it's used